src/react-components/debug-panel/RtcDebugPanel.js

```diff
@@ -0,0 +1,699 @@
+import React, { Component } from "react";
```

Comment:
Anyway we can break this RtcDebugPanel into Left, Right, Middle components?

Reply:
I guess we could, do you think is worth refactoring that even if there are many changes that it's reused?